servicemigration/version.py

```diff
@@ -1,5 +1,6 @@
+import os
 
-API_VERSION = "1.0.0"
+API_VERSION = open(os.path.join(os.path.dirname(__file__), "VERSION"), 'r').readlines()[0]
```

Comment:
The VERSION file is located in the servicemigration package, so I would expect it to be in the image.